### PR TITLE
Getting a powerupobject to spawn

### PR DESCRIPTION
**Pull request by cab0323**
Implementing the PowerUp game object

### CODE REVIEW

**Review by divpatel10 — CHANGES_REQUESTED** [review]
Waiting for the PowerUpSpec and PowerUpSpawnComponent Files, other changes looks great!

**Comment by cab0323** [comment]
Committing the PowerUp classes

File: app/src/main/java/com/example/gauntlet/GameEngine.java [hunk 1]
```diff
@@ -168,7 +166,14 @@ public void deSpawnReSpawn() {
                 .spawn(objects.get(Level.PLAYER_INDEX)
                         .getTransform());
 
-        objects.get(Level.FIRST_ALIEN).spawn(objects.get(Level.PLAYER_INDEX).getTransform());
+
+      objects.get(Level.FIRST_ALIEN).spawn(objects.get(Level.PLAYER_INDEX).getTransform());
```
Comment (divpatel10) [inline app/src/main/java/com/example/gauntlet/GameEngine.java:170]: May need to comment this out after merge

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Wed Jul 07 23:24:32 EDT 2021
+#Thu Jul 08 10:28:53 EDT 2021
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (divpatel10) [inline gradle/wrapper/gradle-wrapper.properties]: Change back to Gradle version 6.5 after merge

**Pull request merged**